src/message.c

```diff
@@ -933,7 +933,7 @@ void qd_message_free(qd_message_t *in_msg)
         // counts starting with the msg cursor.  If the buffer count drops to
         // zero we can free it.
         //
-        LOCK(content->lock);
```

Comment:
Why not add the '&' to the macro rather than having to add '&' to all of the invocations?

Reply:
Done!  Thanks :+1: